site/check_code_indent.py

```diff
@@ -0,0 +1,41 @@
+#  Copyright (C) 2022 Dremio
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import re
+import sys
+from pathlib import Path
+
+if __name__ == '__main__':
```

Comment:
nit: maybe add comments about the purpose of this check?

Reply:
i briefly thought about it but it seemed redundant with the file name to some degree.
also the code is so small (and the printed messages "self-explaining") that i opted to not do this at this time, hope thats ok :sweat_smile:

Reply:
fair enough :)